### ISSUE TITLE
Assign Tester focus not entering the menu when opened

### ISSUE THREAD

**Issue by howard-e** [w3c/aria-at-app#992]
During a [recent CG meeting](https://www.w3.org/2024/03/27-aria-at-minutes.html#t04), it came up that when the Assign Tester button for an admin is pressed and the menu is opened, focus isn't automatically being set to the menu for NVDA, when it should be. JAWS may already be doing this.
